pkg/infra/grn/grn.go

```diff
@@ -0,0 +1,67 @@
+package grn
+
+import (
+	"fmt"
+	"strings"
+)
+
+type GRN struct {
+	// TenantID contains the ID of the tenant (in hosted grafana) or
+	// organization (in other environments) the resource belongs to. This field
+	// may be omitted for global Grafana resources which are not associated with
+	// an organization.
+	TenantID string
```

Comment:
Can we make this an int for now?  that will let us avoid a conversion step -- when/if tenants can be anything other than an int, we can update then